### PR TITLE
@craigspaeth => Add CORS and handle Display fallback

### PR DESCRIPTION
**Pull request by kanaabe**
- Adds CORS headers to be able to embed ad in an iframe
- Rehydrate ad on client-side
- Adds a fallback which is an embedded Sailthru signup form
- Adds basic tracking

Fallback is necessary because of https://developers.facebook.com/docs/instant-articles/troubleshooting#ad-troubleshooting

### CODE REVIEW

**Comment by kanaabe** [comment]
Oops, assigned too early -- this is still missing the Event handler: https://github.com/artsy/force/pull/1890/files

**Comment by kanaabe** [comment]
@craigspaeth no rush -- this is ready for review now! Thanks for helping me through some of the tracking parts! I'm a fan of the setup. 🥇

File: client/apps/display/client.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { DisplayPanel } from '@artsy/reaction-force/dist/Components/Publishing/Display/DisplayPanel'
+import track from 'react-tracking'
+const sd = require('sharify').data
+
+@track(
+  { page: 'Instant Article Display Panel' },
+  { dispatch: (data) => {
+    const { action, ...rest } = data
```
Comment (craigspaeth) [inline client/apps/display/client.jsx]: 👌  nice destructuring action here.

**Comment by craigspaeth** [comment]
Nice! LGTM! Exciting stuff. Feel free to merge when 💚

**Pull request merged**